scripts/build-docker

```diff
@@ -24,17 +27,18 @@ fi
 
 if [ ${ARCH} != "amd64" ]; then
 	DOCKERFILE+=".${ARCH}"
-	TAG+="-${ARCH}"
+	VERSION+="-${ARCH}"
```

Comment:
I was following the presence set by flannel: https://quay.io/repository/coreos/flannel?tab=tags
I guess then we could have `${TAG}:latest-${arch}`.